mlflow/tracking/fluent.py

```diff
@@ -861,7 +869,7 @@ def delete_experiment(experiment_id):
     MlflowClient().delete_experiment(experiment_id)
 
 
-def delete_run(run_id):
+def delete_run(run_id) -> None:
```

Comment:
I'm planning to add `-> None` to functions that don't return anything (e.g. `mllfow.set_tags`)